ember/ember-cli-build.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```

Comment:
holy crap! i guess i'm trolling! i think i copied that from ember-cli website. let's see.

<img width="496" alt="screen shot 2017-03-29 at 2 42 28 pm" src="https://cloud.githubusercontent.com/assets/1066080/24470756/02dab2da-148e-11e7-813e-93730bb42349.png">

# I copied from the sourcemap default section!

you are correct!